lib/repo_tools.pm

```diff
@@ -264,6 +264,8 @@ sub rmt_wizard {
         zypper_call 'in rmt-server';
     }
 
+    my $count = 1;
+  WORKAROUND:
```

Comment:
Good workaround. only one tidy comment, can you please add a retry limit [e.g. 5] or it might go to a infinite loop?

Reply:
I didn't want to do it, but ok ...